cmd/systray/systray.go

```diff
@@ -36,6 +36,8 @@ var (
 	localClient tailscale.LocalClient
 	chState     chan ipn.State // tailscale state changes
 
+	chRebuild chan int // triggers a menu rebuild
```

Comment:
chan struct{} maybe so you don't need to worry the reader about what the int or bool or whatever means?

Reply:
done